Packs/Anomali_ThreatStream/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Anomali ThreatStream",
     "description": "Use Anomali ThreatStream to query and submit threats.",
     "support": "xsoar",
-    "currentVersion": "2.0.31",
+    "currentVersion": "2.0.32",
```

```suggestion
    "currentVersion": "2.1.0",
```

Change also the RN file name